docs/troubleshooting.md

```diff
@@ -1,6 +1,6 @@
 ## Whitehall troubleshooting
 
-You may run into errors while working locally and trying to render pages on the Frontend.
+You may run into errors while working locally and trying to render pages.
```

Comment:
Does the fact that the old docs said "on the Frontend" mean there would be no errors locally when rendering pages on the Admin, or that the troubleshooting steps don't apply to the Admin side?

Reply:
My interpretation of this is that "on the Frontend" means anything that is rendered to any users, including in the admin interface.  This is because the troubleshooting steps aren't specific to problems you'd experience in the old frontend version of the app.